instrumentation/testing/testing.go

```diff
@@ -154,6 +161,57 @@ func (test *Test) End() {
 	test.span.Finish()
 }
 
+func (test *Test) extractTestLoggerOutput() {
+	output := extractTestOutput(test.t)
+	if output == nil {
+		return
+	}
+	outStr := string(*output)
+	var logArray []map[string]interface{}
+	for _, match := range TESTING_LOG_REGEX.FindAllString(outStr, -1) {
+		matches := TESTING_LOG_REGEX.FindStringSubmatch(match)
+
+		if matches[1] == "" {
+			msg := matches[3]
+			if strings.Index(msg, "    ") == 0 {
+				msg = msg[4:]
+			}
+			if len(logArray) > 0 {
+				logItem := logArray[len(logArray)-1]
+				logItem["message"] = logItem["message"].(string) + "\n" + msg
+			}
+		} else {
+			logItem := map[string]interface{}{
+				"file":    matches[1],
+				"line":    matches[2],
+				"message": matches[3],
+			}
+			logArray = append(logArray, logItem)
+		}
+	}
+
+	commonFields := []log.Field{
+		log.String(tags.EventType, tags.LogEvent),
+		log.String(tags.LogEventLevel, tags.LogLevel_VERBOSE),
+		log.String("log.logger", "test.Logger"),
+	}
+	for _, item := range logArray {
+		fields := append(commonFields, log.String(tags.EventMessage, item["message"].(string)))
+		fields = append(fields, log.String(tags.EventSource, fmt.Sprintf("%s:%s", item["file"].(string), item["line"].(string))))
```

Comment:
You can join these two into one `append` operation

Reply:
done